src/Analyzers/Analyzers/test/MinimalStartupTest.cs

```diff
@@ -1,269 +1,303 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
 using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Analyzer.Testing;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Testing;
 
-namespace Microsoft.AspNetCore.Analyzers
-{
-    public class MinimalStartupTest : StartupAnalyzerTestBase
-    {
-        public MinimalStartupTest()
-        {
-            Runner = new AnalyzersDiagnosticAnalyzerRunner(StartupAnalyzer, OutputKind.ConsoleApplication);
-        }
+namespace Microsoft.AspNetCore.Analyzers;
 
-        internal override bool HasConfigure => false;
+public class MinimalStartupTest
```

Comment:
I like the refactor mostly, but what I liked about the original code was that we had a base class with all the test cases that applied to both startup analyzers. Now you will need to remember to add a test to both tests (if applicable). Could you keep a base class that's abstract and just declares what test methods Minimal and Startup should implement?

Reply:
Do you think a comment would suffice 

// Remember to also add to MinimalStartupTest if this is updated.

Reply:
```suggestion
// Remember to also update StartupAnalyzerTest if tests in this type are added or updated.

public class MinimalStartupTest
```